.gitignore

```diff
@@ -5,3 +5,4 @@
 .npmrc
 *.lcov
 .log
+package-lock.json
```

Comment:
Better to add line `package-lock=false` to  .npmrc file. In such it case it even wouldn't be created. Although you don't have to add that file to ignore nowadays